native/App.tsx

```diff
@@ -1,8 +1,13 @@
 import React from "react";
-import NavigationRouter from "./src/feature/navigation/NavigationRouter";
+import { NavigationContainer } from "@react-navigation/native";
+import OnboardingNavigator from "./src/feature/navigation/navigator/OnboardingNavigator";
```

Comment:
Casi
pero esto hay que meterlo en
```suggestion
import OnboardingNavigator from "./src/navigation/navigator/OnboardingNavigator";
```

Reply:
Ya realice los cambios 🙂